testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -619,6 +619,73 @@ class PlatformViewWithContinuousTexture extends PlatformViewScenario {
   }
 }
 
+/// A simple platform view for testing backDropFilter with a platform view in the scene.
+///
+/// Note it is not testing applying backDropFilter on a platform view.
+/// See: https://github.com/flutter/flutter/issues/80766
+class PlatformViewWithOtherBackDropFilter extends PlatformViewScenario {
+  /// Constructs a platform view with continuous texture layer.
+  PlatformViewWithOtherBackDropFilter(PlatformDispatcher dispatcher, String text, { int id = 0 })
+      : super(dispatcher, text, id: id);
+
+  @override
+  void onBeginFrame(Duration duration) {
+    final SceneBuilder builder = SceneBuilder();
+
+    final PictureRecorder recorder = PictureRecorder();
+    final Canvas canvas = Canvas(recorder);
+    // This is just a background picture to make the result more viewable.
+    canvas.drawRect(
+      const Rect.fromLTRB(0, 0, 500, 400),
+      Paint()..color = const Color(0xFFFF0000),
+    );
+    // This rect should look blur due to the backdrop filter below.
+    canvas.drawRect(
+      const Rect.fromLTRB(0, 0, 300, 300),
+      Paint()..color = const Color(0xFF00FF00),
+    );
+    final Picture picture = recorder.endRecording();
+    builder.addPicture(const Offset(0, 0), picture);
+
+    final ImageFilter filter = ImageFilter.blur(sigmaX: 8, sigmaY: 8);
+    builder.pushBackdropFilter(filter);
+
+    final PictureRecorder recorder2 = PictureRecorder();
+    final Canvas canvas2 = Canvas(recorder2);
+    // This circle should not look blur.
+    canvas2.drawCircle(
+      const Offset(200, 100),
+      50,
+      Paint()..color = const Color(0xFF0000EF),
+    );
+    final Picture picture2 = recorder2.endRecording();
+    builder.addPicture(const Offset(100, 100), picture2);
+
+    builder.pop();
+
+    builder.pushOffset(0, 600);
+    _addPlatformViewToScene(builder, id, 500, 500);
+    builder.pop();
+
+    final PictureRecorder recorder3 = PictureRecorder();
+    final Canvas canvas3 = Canvas(recorder3);
+    // Add another picture layer so an overlay UIView is created, which was
+    // the root cause of the original issue.
+    canvas3.drawCircle(
+      const Offset(300, 200),
+      50,
+      Paint()..color = const Color(0xFF0000EF),
+    );
+    final Picture picture3 = recorder3.endRecording();
+    builder.addPicture(const Offset(100, 100), picture3);
+
+    final Scene scene = builder.build();
+    window.render(scene);
+    scene.dispose();
+  }
+}
+
+
```

Comment:
could we add another test with a stack similar to:  (back) picture, pv, picture, filter, pv (front)?  I think the 1st pv won't get the filter, but what about the 2nd picture?